### DISCUSSION TITLE
大家有認真上課嗎

### DISCUSSION THREAD

**Discussion by Jinyuu17** [1101-GS4719-C/Python-Community-GS4719-C#2180] [General]
這個討論區在課堂間新增了許多討論串和留言回復，大家有在聽帥氣的家凱老師上課嗎？
我自己一邊聽老師講課，一邊打這邊的文章，活絡討論區。分享一下我今天的戰績，討論串三篇，留言十則左右。和大家分享一下你們的戰績吧，我看到有位帥哥發了超級多的文章的，老師看到應該會很高興XD。

**Comment by gigitilto** [comment]
我只會發沒人想討論的話題，或是在其他人的討論區下提出一些沒什麼資訊內容的留言。那幾個一直在發文的同學，每一篇都是很有深度的知識分享，真的很厲害，他們在大學期間一定學到了很多東西。

**Reply by chia-kai-chang** [reply]
哪一篇，好好奇唷

**Comment by wcyeee** [comment]
我覺得有很多同學提出了一些有趣的議題，我除了可以分享自己的想法，也可以看到許多同學提出的各種不一樣的想法，讓我覺得收穫很多！也有看到許多同學在分享很多專業厲害的訊息，可以在這邊免費看到各種厲害的資訊整理我覺得很快樂！

**Comment by Tchang323** [comment]
會回答這個問題的人，感覺就沒有認真上課XD

**Comment by lai1008** [comment]
我覺得這裡有很多同學提出了一些有趣的議題，我除了可以分享自己的想法，也可以了解到各種不一樣的想法，讓我覺得收穫很多！

**Comment by 113607514** [comment]
我也很喜歡在這邊看討論串的內容和留言，對我來說就很像在滑社群軟體，不僅如此還有機會加分，除了獲得一些知識，還有很多心靈上的收穫。

**Comment by Boying12** [comment]
真的很開心看到這個討論區變得這麼熱鬧，大家都在積極參與！我自己也在努力跟上帥氣的家凱老師的課程，老師的內容實在太吸引人了，讓我每次上課都充滿期待。今天我一邊認真聽著老師的講解，一邊在這裡活絡討論區，感覺時間過得特別快。

**Comment by zoo100130** [comment]
只有我注意到原來原po上課都在看帥哥嗎

**Comment by NiCL100** [comment]
我也好喜歡聽家凱老師上課，雖然有時候因為聽不懂而聽到有點恍神，但是家凱老師非常親切，上課也會講笑話，或是自己講話講到笑場，超有趣的！
